server/src/main/java/org/apache/iotdb/db/qp/physical/sys/OperateFilePlan.java

```diff
@@ -84,6 +103,8 @@ public String toString() {
         + autoCreateSchema
         + ", sgLevel="
         + sgLevel
+        + ", metadataCheck="
```

Comment:
```suggestion
        + ", verify="
```

This should be consistent to user

Reply:
already fix it